Patches/Mincho, The Mint Choco Slime/ThingDefs_Races/Race_Mincho.xml

```diff
@@ -0,0 +1,110 @@
+				<xpath>/Defs/Garam_RaceAddon.RaceAddonThingDef[defName="Mincho_ThingDef"]/comps</xpath>
+				<nomatch Class="PatchOperationAdd">
+					<xpath>/Defs/Garam_RaceAddon.RaceAddonThingDef[defName="Mincho_ThingDef"]</xpath>
+					<value>
+						<comps/>
+					</value>
+				</nomatch>
+				<match Class="PatchOperationAdd">
+					<xpath>/Defs/Garam_RaceAddon.RaceAddonThingDef[defName="Mincho_ThingDef"]</xpath>
+					<value>
+					</value>
+				</match>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs/Garam_RaceAddon.RaceAddonThingDef[defName="Mincho_ThingDef"]/comps</xpath>
+				<value>
+					<li>
+					<compClass>CombatExtended.CompPawnGizmo</compClass>
+					</li>
+					<li Class="CombatExtended.CompProperties_Suppressable" />
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/Garam_RaceAddon.RaceAddonThingDef[defName="Mincho_ThingDef"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Humanoid</bodyShape>
+					</li>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+			 <xpath>/Defs/Garam_RaceAddon.RaceAddonThingDef[defName="Mincho_ThingDef"]/statBases</xpath>
+				  <value>
+					<Suppressability>0.8</Suppressability>
+					<!--Innate immunity to toxic fallout, high immunity gain rate-->
+					<SmokeSensitivity>0</SmokeSensitivity>
+				  </value>
+			</li>
+			
+			<!-- -->
+			
+			<li Class="PatchOperationReplace">
+			 <xpath>/Defs/Garam_RaceAddon.RaceAddonThingDef[defName="Mincho_ThingDef"]/statBases/ArmorRating_Sharp</xpath>
+				  <value>
+					<ArmorRating_Sharp>1</ArmorRating_Sharp>
+				  </value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+			 <xpath>/Defs/Garam_RaceAddon.RaceAddonThingDef[defName="Mincho_ThingDef"]/statBases/ArmorRating_Blunt</xpath>
+				  <value>
+					<ArmorRating_Blunt>0.5</ArmorRating_Blunt>
+				  </value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/Garam_RaceAddon.RaceAddonThingDef[defName="Mincho_ThingDef"]/tools</xpath> 
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>slime fist</label>
+							<capacities>
+								<li>Blunt</li>
+								<li>Poke</li>
+								<li>Cut</li>
+							</capacities>
+							<power>1</power>
+							<cooldownTime>0.9</cooldownTime>
+							<linkedBodyPartsGroup>Torso</linkedBodyPartsGroup>
+							<armorPenetrationSharp>0.05</armorPenetrationSharp>
+							<armorPenetrationBlunt>0.25</armorPenetrationBlunt>
+						</li>
+					</tools>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+			 <xpath>/Defs/Garam_RaceAddon.RaceAddonThingDef[defName="Mincho_ThingDef"]/raceAddonSettings/apparelRestrictionSetting/allAllow_Exceptions</xpath>
+				  <value>
+					<li>Apparel_TacVest</li>
+					<li>Apparel_Backpack</li>
+					<li>Apparel_TribalBackpack</li>
+					<li>Apparel_BallisticShield</li>
+					<li>Apparel_MeleeShield</li>
+					<li>Apparel_GasMask</li>
```

Comment:
Do they really need to be able to wear gas masks of any variety if they don't get smoke inhalation?